ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml

```diff
@@ -5,8 +5,12 @@ ref:
     name: ocp-qe-perfscale-ci
     tag: latest
   env:
+  - name: E2E_VERSION
+    default: "default"
```

Comment:
I believe this is the version we want to hard-code to lock-in the specific e2e tag? "default" is the same as previous "latest", which leaves us open to unexpected changes.

Reply:
Yes it picks `latest` tag by default. But before we even cut a release with `latest` tag, it is recommended to test and verify them using `pre-release` version and once everything works fine we can get it tagged as `latest`.
Example: https://github.com/vishnuchalla/e2e-benchmarking/releases/tag/v0.0.0

Reply:
I don't think using latest solves the problem. Please hard-code the exact release version we've tested.

Reply:
My idea was to use latest tag by default, and pass on a rule to the team as _"whenever some is cutting a release go with the pre-release first, verify it in PROW, and if it works well mark it as latest"_. If we think this plan doesn't work, we can hard code the tag number here. But one thing to note with hard-coding is, for every new release we will need to raise a PR here updating the value of `E2E_VERSION` env in multiple files, whereas it is not required in the above approach that I am suggesting.

Reply:
Ok. I think that's a process we can refine as needed.

> But one thing to note with hard-coding is, for every new release we will need to raise a PR here updating the value of E2E_VERSION env in multiple files,

This is very much the intent for now: To freeze the e2e version and require a PR and human eyes/hands on the upgrade step so nothing catches us by surprise.

Thanks!

Reply:
Done.